Code/src/main/java/nl/utwente/viskell/ui/CircleMenu.java

```diff
@@ -22,11 +24,23 @@
  */
 public class CircleMenu extends CirclePopupMenu {
 
+    /** List of all block on which a CircleMenu is active. */
+    private static ArrayList<Block> activeBlocks = new ArrayList<>();
```

Comment:
While I can't come up with any ways it could actually go wrong, I have a bit of a dislike for global, mutable state like this. Is there a better way to store this? Maybe a block could keep track of whether it has a circlemenu on it or not?